### PR TITLE
Review Subcommand

### PR DESCRIPTION
**Pull request by brhoades**
This subcommand allows you to walk through all student submissions alphabetically. It uses an editor call from your gradesheet's assignment.yml. Defaults to calling vim like so:

`/usr/bin/vim -O2 {0} {1} {2} {3}`

Which opens each assignment in two windows. The 1st window, {0}, contains the first file from the student's repository. The 2nd is the latest result.log. The 2nd and 3rd are the remaining files for student files and results respectively.

After viewing is finished, it tells you the next assignment, how many are remaining, and provides a menu. The menu allows you to quit, continue, or go back.

I use this when grading submissions as I print nothing out, and it allows me to walk through things in the order of their gradesheets.

Looking for feedback.

### CODE REVIEW

**Comment by michaelwisely** [comment]
I don't think I quite understand what `{0}` through `{3}` correspond to.

Also, what do you mean by "[{0}]contains the first file from the student's repository"? "first" as in "first alphabetically"?

And is "repository" a general term for lump of code? We shouldn't expect that all imported submissions will be git repos.

**Comment by LinuxMercedes** [comment]
@brhoades What do you think about adding a feature so review picks up at the last assignment you were grading?

**Comment by brhoades** [comment]
@LinuxMercedes works for me, although I'd say let's put that in another PR and finish this one. That feature should come with some sort of user state storage.

**Comment by LinuxMercedes** [comment]
Yep, that seems reasonable. Let's not merge review just yet--there are a couple little tweaks I want to make to it first. I'll try to do those early next week.

**Comment by brhoades** [comment]
Good call on sorted. Looks good from there.

**Comment by LinuxMercedes** [comment]
@brhoades if you'd like to give my changes a once-over, I'd appreciate it. Most crucially, there are now menu options to regrade an assignment or to inspect an assignment container.  It's quite slick for 5201 style grading.

**Comment by brhoades** [comment]
:ship: :ship: :ship: :ship: :ship: :ship:

File: grader/grader/commands/review.py [hunk 1]
```diff
@@ -63,15 +65,21 @@ def review_loop(assignment, start_at):
             print("{0} is next.\n".format(user_ids[i+1]))
 
             choice = "A"
-            while choice.upper() not in ["C", "Q", "P", "L", "U", ""]:
+            while choice.upper() not in ["C", "Q", "P", "L", "U", "I", "G", "N", "H", "R", ""]:
```
Comment (brhoades) [inline grader/grader/commands/review.py]: X, Y, and Z!

File: grader/grader/commands/review.py [hunk 2]
```diff
@@ -63,15 +65,21 @@ def review_loop(assignment, start_at):
...
                 print("C|N)   Continue (default)")
                 if i > 0:
                     print("H|P)   Previous")
+                print("R)     Review current student's submission")
+                print("I)     Inspect current student's assignment container")
```
Comment (brhoades) [inline grader/grader/commands/review.py]: This was my dream

**Pull request merged**